src/memvid/encoder.py

```
@@ -0,0 +1,169 @@
+        self.chunks.extend(chunks)
+        logger.info(f"Added {len(chunks)} chunks. Total: {len(self.chunks)}")
+    
+    def add_text(self, text: str, chunk_size: Optional[int] = None, overlap: Optional[int] = None):
+        """Add text and automatically chunk it"""
+        chunk_size = chunk_size or self.config["chunking"]["chunk_size"]
+        overlap = overlap or self.config["chunking"]["overlap"]
+        
+        chunks = chunk_text(text, chunk_size, overlap)
+        self.add_chunks(chunks)
+    
+    def build_video_simple(self, output_path: str, index_path: str) -> bool:
```

Pylint (3.3.7)

[refactor] Too many local variables (18/15)

(R0914)